recipes-python/pycoral/pycoral_2.0.0.bb

```diff
@@ -24,7 +24,7 @@ DEPENDS = " \
 PATCHTOOL = "git"
 
 S = "${WORKDIR}/git"
-SRCREV = "9972f8ec6dbb8b2f46321e8c0d2513e0b6b152ce"
+SRCREV = "ae743b0b234d66d784826ccfbb4607a69dc1a2a8"
```

Comment:
Is that still version 2.0.0 of pycoral?

Reply:
This is 2.0.0 + two fixes in the examples, which could be relevant for us. Unfortunately no new release yet. Either we release that as 2.0.0+20220505-1, or we simply stick to the release and drop this commit.

Reply:
Dropped this commit. The added value is too low to move away from a release.